File: ITP1/2/c.js
var inputs = require('fs').readFileSync('/dev/stdin', 'utf8').split(" ");

a = ~~inputs[0];
b = ~~inputs[1];
c = ~~inputs[2];

x = Math.min(a,b,c)
z = Math.max(a,b,c)
y = a+b+c-x-z
console.log(`${x} ${y} ${z}`);

File: ITP1/2/c.cpp
#include <iostream>
#include <stdio.h>
using namespace std;


int main(){
    int a, b, c, x, y, z;
    cin >> a >> b >> c;
    x = min(min(a, b), c);
    z = max(max(a, b), c);
    y = a+b+c-x-z;
    printf("%d %d %d\n",x,y,z);
    return 0;
}
